api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -36,7 +36,10 @@ message VmConfig {
   // The Wasm code that Envoy will execute.
   config.core.v3.AsyncDataSource code = 3;
 
-  // The Wasm configuration used in initialization of a new VM (proxy_on_start).
+  // The Wasm configuration used in initialization of a new VM
+  // (proxy_on_start). `google.protobuf.Struct` is serialized as JSON before
```

Comment:
Let's use name `proxy_on_vm_start` instead of `proxy_on_start ` for consistency with https://github.com/proxy-wasm/spec/tree/master/abi-versions/vNEXT

Reply:
Can we leave the naming for later? It's guaranteed to churn endlessly and will only freeze when the code lands.

Reply:
Sure